Languages/IronPython/Tests/compat/common.py

```diff
@@ -150,8 +150,8 @@ def get_platform_string(current = None):
     if current == None:
         import sys
         current = sys.platform
-        
-    if current.startswith("win") or current.startswith("linux"): return "nat"
+    
+    if [True for x in ["win", "linux", "darwin"] if current.startswith(x)]: return "nat"
```

Comment:
Might be more readable if it was something like
`any(current.startswith(x) for x in ...)`